modules/programs/autorandr.nix

```diff
@@ -114,6 +114,54 @@ let
           for the documentation of the transform matrix.
         '';
       };
+
+      dpi = mkOption {
+        type = types.nullOr types.int;
```

Comment:
If you feel like being fancy then you can use `types.ints.positive` instead of `types.int`.